common/src/test/java/com/microsoft/identity/common/nativeauth/util/ApiConstants.kt

```diff
@@ -29,17 +29,17 @@ import java.net.URL
  */
 interface ApiConstants {
     companion object {
-        val signUpStartRequestUrl = URL("https://native-auth-mock-api-private-preview.azurewebsites.net/1234/signup/v1.0/start")
-        val signUpChallengeRequestUrl = URL("https://native-auth-mock-api-private-preview.azurewebsites.net/1234/signup/v1.0/challenge")
-        val signUpContinueRequestUrl = URL("https://native-auth-mock-api-private-preview.azurewebsites.net/1234/signup/v1.0/continue")
-        val signInInitiateRequestUrl = URL("https://native-auth-mock-api-private-preview.azurewebsites.net/1234/oauth2/v2.0/initiate")
-        val signInChallengeRequestUrl = URL("https://native-auth-mock-api-private-preview.azurewebsites.net/1234/oauth2/v2.0/challenge")
-        val signInTokenRequestUrl = URL("https://native-auth-mock-api-private-preview.azurewebsites.net/1234/oauth2/v2.0/token")
-        val ssprStartRequestUrl = URL("https://native-auth-mock-api-private-preview.azurewebsites.net/1234/resetpassword/v1.0/start")
-        val ssprChallengeRequestUrl = URL("https://native-auth-mock-api-private-preview.azurewebsites.net/1234/resetpassword/v1.0/challenge")
-        val ssprContinueRequestUrl = URL("https://native-auth-mock-api-private-preview.azurewebsites.net/1234/resetpassword/v1.0/continue")
-        val ssprSubmitRequestUrl = URL("https://native-auth-mock-api-private-preview.azurewebsites.net/1234/resetpassword/v1.0/submit")
-        val ssprPollCompletionRequestUrl = URL("https://native-auth-mock-api-private-preview.azurewebsites.net/1234/resetpassword/v1.0/poll_completion")
+        val signUpStartRequestUrl = URL("https://native-auth-mock-api.azurewebsites.net/1234/signup/v1.0/start")
+        val signUpChallengeRequestUrl = URL("https://native-auth-mock-api.azurewebsites.net/1234/signup/v1.0/challenge")
+        val signUpContinueRequestUrl = URL("https://native-auth-mock-api.azurewebsites.net/1234/signup/v1.0/continue")
+        val signInInitiateRequestUrl = URL("https://native-auth-mock-api.azurewebsites.net/1234/oauth2/v2.0/initiate")
+        val signInChallengeRequestUrl = URL("https://native-auth-mock-api.azurewebsites.net/1234/oauth2/v2.0/challenge")
+        val signInTokenRequestUrl = URL("https://native-auth-mock-api.azurewebsites.net/1234/oauth2/v2.0/token")
+        val ssprStartRequestUrl = URL("https://native-auth-mock-api.azurewebsites.net/1234/resetpassword/v1.0/start")
+        val ssprChallengeRequestUrl = URL("https://native-auth-mock-api.azurewebsites.net/1234/resetpassword/v1.0/challenge")
+        val ssprContinueRequestUrl = URL("https://native-auth-mock-api.azurewebsites.net/1234/resetpassword/v1.0/continue")
+        val ssprSubmitRequestUrl = URL("https://native-auth-mock-api.azurewebsites.net/1234/resetpassword/v1.0/submit")
+        val ssprPollCompletionRequestUrl = URL("https://native-auth-mock-api.azurewebsites.net/1234/resetpassword/v1.0/poll_completion")
```

Comment:
Looks like these changes are reverting the recently made changes from the dev branch - is this intended?

Reply:
Yes the updated mock api lives in a different url than the private preview one

Reply:
@GBakolas so what is the mock API strategy here? Stable features live on `https://native-auth-mock-api.azurewebsites.net` - what lives on the `-private-preview` URL version?

Reply:
Yes the production/public state of the API should be on the https://native-auth-mock-api.azurewebsites.net domain. We will use the second domain for testing new features, although I think we should change the name to something more suitable. This was the initial idea but we can discuss this further with the rest of the team